definitions/tools/docm_gatk_haplotype_caller.cwl

```diff
@@ -2,47 +2,59 @@
 
 cwlVersion: v1.0
 class: CommandLineTool
-label: "HaplotypeCaller (GATK 3.6)"
-baseCommand: ["/usr/bin/java", "-Xmx8g", "-jar", "/opt/GenomeAnalysisTK.jar", "-T", "HaplotypeCaller"]
+label: "HaplotypeCaller (GATK4)"
+baseCommand: ["/bin/bash", "docm_haplotypeCaller.sh"]
 requirements:
     - class: ResourceRequirement
       ramMin: 9000
     - class: DockerRequirement
-      dockerPull: "mgibio/gatk-cwl:3.6.0"
-arguments:
-    ["-gt_mode", "GENOTYPE_GIVEN_ALLELES",
-    "-o", { valueFrom: $(runtime.outdir)/docm_raw_variants.vcf }]
+      dockerPull: "broadinstitute/gatk:4.1.2.0"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'docm_haplotypeCaller.sh'
+        entry: |
+             set -o pipefail
+             set -o errexit
+
```

Comment:
To help the person who has to debug this someday (and that person might be you 😆 ), can you explicitly capture variables into names?   

reference=$1
vcf=$2   

(or whatever - I made those up)

Reply:
Changed the file